LEGO1/mxtransitionmanager.h

```diff
@@ -2,19 +2,58 @@
 #define MXTRANSITIONMANAGER_H
 
 #include "mxcore.h"
+#include "mxvideopresenter.h"
+#include "legovideomanager.h"
+#include "legoomni.h"
 
-class MxVideoPresenter;
 
-// 0x100d7ea0
+// VTABLE 0x100d7ea0
 class MxTransitionManager : public MxCore
 {
 public:
-  MxTransitionManager();
-  virtual ~MxTransitionManager() override; // vtable+0x0
+    MxTransitionManager();
+    virtual ~MxTransitionManager() override; // vtable+0x0
 
-  __declspec(dllexport) void SetWaitIndicator(MxVideoPresenter *videoPresenter);
+    __declspec(dllexport) void SetWaitIndicator(MxVideoPresenter *videoPresenter);
 
-  virtual MxResult Tickle(); // vtable+0x8
+    virtual MxResult Tickle(); // vtable+0x8
```

Comment:
We are currently using 2-space indents in the project as per the Styleguide, could you update the code? We should probably introduce a linter (and a check in the CI) so stuffokes that becomes easier to handle but for now we have to do it manually.

Reply:
Sorry about that, I wasn't aware. Maybe this should be added to the style guide in CONTRIBUTING.MD?